programs/util.c

```diff
@@ -326,6 +326,27 @@ int UTIL_prepareFileList(const char *dirName, char** bufStart, size_t* pos, char
 
 #endif /* #ifdef _WIN32 */
 
+/* Check if the file is precompressed (.zst, .lz4, .gz, .xz).
+YES => Skip the file (return 0)
+NO => return 1
+*/
+int UTIL_isPrecompressedFile(const char *inputName)
```

Comment:
Considering the name of the function `isPrecompressedFile()`,
it feels more logical to return `1` (==true) when the file is indeed compressed.

Also, maybe `isCompressedFile()` is enough.
Not sure what `Pre` in `Precompressed` implies.

Reply:
Resolved in latest commit "Fixing tests and moving flag to advanced usage 0e9a37d"